pkgs/applications/blockchains/bitcoin.nix

```diff
@@ -26,7 +26,7 @@ in stdenv.mkDerivation rec {
     urls = [ "https://bitcoincore.org/bin/bitcoin-core-${version}/bitcoin-${version}.tar.gz"
              "https://bitcoin.org/bin/bitcoin-core-${version}/bitcoin-${version}.tar.gz"
            ];
-    sha256 = "7ac9f972249a0a16ed01352ca2a199a5448fe87a4ea74923404a40b4086de284";
+    sha256 = "1h3w7brc18145np920vy7j5ms5hym59hvr40swdjx34fbdaisngj";
```

Comment:
Can you replace this with the hex version of the sha256 value?  This makes it easier to compare against the digitally signed value made by the maintainers.

Thanks.